### ISSUE TITLE
Problem To install

### ISSUE THREAD

**Issue by mahyariiii** [afiniel/Yiimp-Pool-Installer#2]
I Have 3 VPS Ubuntu 18.04
I tried to install it several times, but after the initial installation and after the restart
The buttons do not work when installing yiimp version 4.7.3
I tried to reinstall the operating system a few times to make sure, but again it was the same problem.
But when I select yiimp with another developer version 2 , it installs.
Please check and fix this bug.
I'm very happy to use your up-to-date program.

**Comment by afiniel** [comment]
I will have a look at it now, Do you have any logs or screenshot to share

**Comment by mahyariiii** [comment]
@afiniel 
![Untitled](https://user-images.githubusercontent.com/94556716/164688206-9366fd6e-612f-4acd-983e-5e2b0e749a76.jpg)


![1](https://user-images.githubusercontent.com/94556716/164688217-cc51d120-773b-49dc-b420-a893952f8ccc.jpg)

**Comment by afiniel** [comment]
Ah it use incorrect Tag v4.7.1 i have recently added the multi server but i get back soon with Update, the installer need update on the Sql dump schema aswell

**Comment by mahyariiii** [comment]
Note when downloading the written version v4.7.3
When installing in the photo above 4.7.2
Shows 4.7.1 when loading for download and installation

**Comment by mahyariiii** [comment]
> Ah it use incorrect Tag v4.7.1 i have recently added the multi server but i get back soon with Update, the installer need update on the Sql dump schema aswell

thanks  I'm waiting for your update

**Comment by afiniel** [comment]
Thank you for the report

**Comment by mahyariiii** [comment]
Hi 
When installing 
The following errors are displayed
Ubuntu version 18.04

..........................................

![Untitled 1](https://user-images.githubusercontent.com/94556716/164913643-b1a60db5-34e7-4cd1-8434-e6fa494e1178.jpg)

...........................................

![Untitled](https://user-images.githubusercontent.com/94556716/164913645-c164ef1c-529c-4585-852c-4598cd914756.jpg)


...........................................

**Comment by mahyariiii** [comment]
After install web site    -    Unable to connect

**Comment by mahyariiii** [comment]
in the /var/log/letsencrypt     
 certbot.errors.FailedChallenges: Failed authorization procedure. localhost (http-01): urn:ietf:params:acme:error:connection :: The server could not connect to the client to verify the domain

**Comment by afiniel** [comment]
Hi sry o have been busy. But the installer is now working and i have move the project to here: https://github.com/Afiniel-tech/Yiimpool-Installer But the fontend got wrong color and its hard to seee text. i fix that tomorrow. but you can still install the yiimp and set text color to Grey/black manually in your config when the install is completed. So tomorrow or later tonight i fix the last on fontend and it will come more algos tomorrow aswell. Enyoj and Happy mining mate! Regards Afiniel

PS here you see what i mean with the incorrect 
![bild](https://user-images.githubusercontent.com/44875524/165093833-a3c685d6-ac94-4d63-8099-bd756401b861.png)
![bild](https://user-images.githubusercontent.com/44875524/165094157-d6343891-02c2-49ac-ba14-e0937c063f16.png)

**Comment by afiniel** [comment]
@mahyariiii  :)

**Comment by mahyariiii** [comment]
hi thank you 
i  installing  (https://github.com/Afiniel-tech/Yiimpool-Installer)  version yiimp without wingard  
Installed correctly
The font color was right for me
The color problem is probably due to a site template problem
I try to install a few coins and check the connection to the nodes and the coin mining

**Comment by mahyariiii** [comment]
![daemoonbuilder problem](https://user-images.githubusercontent.com/94556716/165103697-a6315430-f52d-418b-a027-88d973e91573.jpg)

**Comment by mahyariiii** [comment]
daemon builder is not fix
Until the problem is resolved
I try to install coins directly on the root

**Comment by afiniel** [comment]
Oh forgot to set the daemon public I’m not home atm but I did it via phone

**Comment by mahyariiii** [comment]
Does not work
Most parts have problems with the site template
Other sections of the coin settings are empty -
.................................................................................................................................................................
![add coin blunk](https://user-images.githubusercontent.com/94556716/165108377-27f1bd18-38e5-4531-a249-62cc1d1775a7.jpg)
.

**Comment by afiniel** [comment]
Yeah I have update that on discord I will fix the fontend stuff  as fast I get home

**Comment by mahyariiii** [comment]
Thank you, I'm waiting
I have a very fast VPS to test, I just rented it for testing
VPS with 16 GB of RAM and 500 GB of nvme
